napalm_logs/listener/kafka.py

```diff
@@ -30,12 +29,13 @@ class KafkaListener(ListenerBase):
     '''
     Kafka listener class.
     '''
-    def __init__(self, address, port, pipe):
-        self.address = address
-        self.port = port
+    def __init__(self, address, port, pipe, **kwargs):
         self.pipe = pipe
-        self.topic = KAFKA_LISTENER_TOPIC
```

Comment:
We should have an explicit topic -- if not provided by the user in the opts, we should use the default. Depending on the usecase of the transport, we can set this inside the base class, to avoid any complexity.